config/routes.rb

```diff
@@ -1,3 +1,4 @@
 Rails.application.routes.draw do
-  # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
+  root "monsters#index"
+  resources :monsters
```

Comment:
All the RESTful routes? Are you sure you don't want to restrict these down to just the ones you're using?